src/main/webapp/WEB-INF/tags/instructor/courseStudentDetailsEdit/studentInformationTable.tag

```diff
@@ -10,7 +10,7 @@
         <form action="<%=Const.ActionURIs.INSTRUCTOR_COURSE_STUDENT_DETAILS_EDIT_SAVE%>" method="post" class="form form-horizontal">
             <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="${studentInfoTable.course}">
             <div class="form-group">
-                <label class="col-sm-1 control-label">Student Name:</label>
+                <label class="col-sm-1 control-label">Name:</label>
```

Comment:
For these kind of unrelated things, open a new issue and tackle it instead -

1. It bloats up the current PR
2. Makes the PR have conflict with the base much more easily which delays the collaboration process